source/databases-collections/time-series.txt

```diff
@@ -0,0 +1,182 @@
+.. _scala-time-series:
+
+================
+Time Series Data
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-short+} to store
+and interact with **time series data**.
+
+Time series data is composed of the following components:
+
+- Measured quantity
+- Timestamp for the measurement
+- Metadata that describes the measurement
+
+The following table describes sample situations for which you could store time
+series data:
+
+.. list-table::
+   :widths: 33, 33, 33
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Situation
+     - Measured Quantity
+     - Metadata
+
+   * - Recording monthly sales by industry
+     - Revenue in USD
+     - Company, country
+
+   * - Tracking weather changes
+     - Precipitation level
+     - Location, sensor type
+
+   * - Recording fluctuations in housing prices
+     - Monthly rent price
+     - Location, currency
+
+.. _scala-time-series-create:
+
+Create a Time Series Collection
+-------------------------------
+
+.. important:: Server Version for Time Series Collections
+
+   To create and interact with time series collections, you must be
+   connected to a deployment running {+mdb-server+} 5.0 or later.
+
+You can create a time series collection to store time series data.  
+To create a time series collection, pass the following parameters to the 
+``createCollection()`` method:
+
+- The name of the new collection to create
+
+- A `CreateCollectionOptions <{+api+}/org/mongodb/scala/model/package$$CreateCollectionOptions$.html>`__
```

Comment:
S: it looks a little unnatural, but I'd take out the articles based on the [style guide](https://www.mongodb.com/docs/meta/style-guide/style/lists/#list-items)
```suggestion
- Name of the new collection to create

- `CreateCollectionOptions <{+api+}/org/mongodb/scala/model/package$$CreateCollectionOptions$.html>`__
```